src/Application/Characters/Models/CharacterPublicViewModel.cs

```diff
@@ -1,10 +1,14 @@
+using Crpg.Application.Common.Mappings;
+using Crpg.Application.Users.Models;
 using Crpg.Domain.Entities.Characters;
 
 namespace Crpg.Application.Characters.Models;
 
-public record CharacterPublicViewModel
+public record CharacterPublicViewModel: IMapFrom<Character>
 {
-    public int Id { get; init; }
-    public int Level { get; init; }
-    public CharacterClass Class { get; init; }
+        public int Id { get; init; }
+        public int Level { get; init; }
+        public CharacterClass Class { get; init; }
+        public CharacterRatingViewModel? Rating { get; init; }
+        public UserPublicViewModel? User { get; init; }
```

Comment:
Those should not be nullable

Reply:
they're not anymore.
Also i added region because orle asked me